internal/cluster/common.go

```diff
@@ -35,6 +35,7 @@ const (
 	statusInfoPreparingForInstallationTimeout = "Preparing cluster for installation timeout"
 	statusInfoPendingForInput                 = "User input required"
 	statusInfoError                           = "cluster has hosts in error"
+	statusInfoTimeout                         = "cluster installation timed out while pending user action"
```

Comment:
a really slow installation cannot fail on it as well?
or some other step will timeout before the general timeout

Reply:
Other steps will timeout and fail the installation.
TMK pending user action is the only status that allows the installation to hang without a timeout.